index.js

```diff
@@ -101,9 +101,10 @@ function WebTorrent (opts) {
     if (opts.wrtc) {
       // TODO: remove in v1
       console.warn('WebTorrent: opts.wrtc is deprecated. Use opts.tracker.wrtc instead')
-      self.tracker.wrtc = opts.wrtc // to support `webtorrent-hybrid` package
+      self.tracker.wrtc = opts.wrtc
```

Comment:
Good call removing this comment -- this option can be used by any package, not just `webtorrent-hybrid` :)